nautobot/docs/administration/nautobot-server.md

```diff
@@ -383,10 +383,10 @@ Done.
 
 Run a job (script, report) to validate or update data in Nautobot.
 
-`--commit`<br>
+`--commit`
```

Comment:
Should these (here and below) also have trailing `  ` to replace the `<br>`?

Reply:
Yes they do.  Added.